src/argus_htmx/views.py

```diff
@@ -65,5 +71,41 @@ def incident_add_ack(request, pk: int, group: Optional[str] = None):
     return render(request, "htmx/incidents/incident_add_ack.html", context=context)
 
 
+class HtmxHttpRequest(HttpRequest):
+    htmx: HtmxDetails
+
+
+@require_GET
+def incidents_table(request: HtmxHttpRequest) -> HttpResponse:
+    # Load incidents
+    qs = Incident.objects.all().order_by("-start_time")
+    latest = qs.latest("start_time").start_time
+
+    # Standard Django pagination
+    page_num = request.GET.get("page", "1")
+    page = Paginator(object_list=qs, per_page=10).get_page(page_num)
+
+    # The htmx magic - use a different, minimal base template for htmx
+    # requests, allowing us to skip rendering the unchanging parts of the
+    # template.
+    if request.htmx:
+        base_template = "htmx/incidents/_table.html"
+    else:
+        base_template = "htmx/incidents/_base.html"
```

Comment:
How will this work with filters? We could add just one as a placeholder. Acked, say, adding more stuff to `request.GET`.